symbolicatorprocessor/symbolicator_test.go

```diff
@@ -41,9 +53,18 @@ func TestSymbolicator(t *testing.T) {
 func TestSymbolicatorCache(t *testing.T) {
 	ctx := context.Background()
 
+	testTel := componenttest.NewTelemetry()
+	tb, err := metadata.NewTelemetryBuilder(testTel.NewTelemetrySettings())
+	assert.NoError(t, err)
+	defer tb.Shutdown()
+
+	attributes := attribute.NewSet(
+		attribute.String("processor_type", "symbolicator"),
```

Comment:
We could probably get more specific about this in the future since this is the source map symbolicator (vs proguard vs dsym).